src/sentry/integrations/pagerduty/utils.py

```diff
@@ -15,6 +16,7 @@
 def build_incident_attachment(action, incident, integration_key, metric_value=None, method=None):
     data = incident_attachment_info(incident, metric_value, action=action, method=method)
     incident_status = incident_status_info(incident, metric_value, action=action, method=method)
+    severity = ""
```

Comment:
Is this one of those unboundvariable errors?

Reply:
Yeah, I couldn't commit without handling it.